packages/demo-nextjs/src/lib/db/item/getItems.ts

```diff
@@ -6,33 +6,47 @@ import { Item } from "../types";
 import { toItemConvertor } from "./utils";
 
 type Query = {
-  type: string;
+  type?: string;
```

Comment:
Need to create 2 types something like this
![image](https://github.com/user-attachments/assets/b384681f-69da-4b75-b50d-de383d49162c)